pkg/networkmanager/index.html

```diff
@@ -108,56 +108,92 @@ <h2 class="panel-title">
         <div id="networking-firewall-summary" class="panel-body">
         </div>
       </div>
-      <div class="panel panel-default"
-           id="networking-interfaces">
-        <div class="panel-heading">
-          <h2 class="panel-title" translate="yes">Interfaces</h2>
-          <div class="panel-actions">
-            <button translate="yes" id="networking-add-bond" class="btn btn-default network-privileged">Add Bond</button>
-            <button translate="yes" id="networking-add-team" class="btn btn-default network-privileged">Add Team</button>
-            <button translate="yes" id="networking-add-bridge" class="btn btn-default network-privileged">Add Bridge</button>
-            <button translate="yes" id="networking-add-vlan" class="btn btn-default network-privileged">Add VLAN</button>
+    </div>
```

Comment:
Seems this end tag is here harmful. Due to it all the following panels are not under `container-fluid` class and it breaks the page.

Reply:
I have fixed this in the latest commits. There were a cople closing div tags out of place